### PR TITLE
French translation added to locales

### PR DESCRIPTION
**Pull request by PaulDach**

### CODE REVIEW

**Comment by funkydude** [comment]
There's also

```
LegionInvasionTimer.toc ## Notes
Options\Locales\frFR.lua
Options\LegionInvasionTimer_Options.toc ## Notes
```

**Comment by PaulDach** [comment]
@funkydude : It's done :)

File: Locales/frFR.lua [hunk 1]
```diff
@@ -4,31 +4,31 @@ local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Nouvelles invasions disponibles !"
+L.tooltipClick = "|cffeda55fClick|r pour glisser et déplacer."
+L.tooltipClickOptions = "|cffeda55fRight-Click|r pour ouvrir les options."
```
Comment (funkydude) [inline Locales/frFR.lua]: This is missing the translation for "right-click". 

What about the remaining English words, did you struggle to translate them?

Comment (PaulDach) [reply Locales/frFR.lua]: We can use English for clicks but I'll translate them if you want.
About the English words, some do not have translation in French but they are used by the game so it's not a problem :)

**Comment by funkydude** [comment]
I just noticed you're using a fake email address in your Git client, please do not do this. If you wish to keep your email address private, go into your GitHub settings, emails > and tick private mode. After doing that, go into your Git client and change the email address to `PaulDach@users.noreply.github.com`

This way you don't break commit history.

**Comment by PaulDach** [comment]
Yes my bad, sorry.
It's fixed now

**Comment by funkydude** [comment]
I believe you will need to commit something new, any change really.

**Comment by PaulDach** [comment]
Tell me if it's ok now.

**Comment by funkydude** [comment]
Can you add the translation of this at the bottom of the french file, thanks
https://github.com/funkydude/LegionInvasionTimer/blob/master/Locales/enUS.lua#L35

**Pull request merged**